src/main/webapp/team.html

```diff
@@ -47,7 +47,7 @@
 
 <section class="section pb-0" style="background-color: #f8fdff">
     <div class="container">
-        <p class="display-3 text-center">Advisory Panel</p>
+        <p class="display-3 text-center" style="margin-bottom: 80px">Advisory Panel</p>
```

Comment:
Use bootstrap's `mb` class instead of using inline CSS. 

`<p class="display-3 text-center mb-5">... `